Remove Article Y, Add 2a4 (#1042)
thewca/wca-regulations: wca-regulations.md

## <article-2><competitors><competitors> Article 2: Competitors

- 2a) Any person may compete in a WCA competition if they:
    - 2a1) Comply with WCA Regulations.
    - 2a2) Meet the competition requirements, which must be clearly announced before the competition.
    - 2a3) Are not suspended by the WCA Board.

>     - 2a4) Comply with all reasonable safety measures set forth by local governments and have it be announced clearly before the competition.

Reviewer: ```suggestion
    - 2a4) Comply with all reasonable safety measures announced clearly before the competition which are set by local governments.
```

Reviewer: It sounds more natural to me.

>     - 2a4) Comply with all reasonable safety measures set forth by local governments and have it be announced clearly before the competition.

Reviewer 2: ```suggestion
    - 2a4) Comply with all reasonable safety measures deemed necessary by the WCA Delegate, which must be clearly announced before the competition.
```

New proposed wording after WCA Staff feedback.
